### PR TITLE
Add inverted prop

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Add inverted prop.

#### What problem is this solving?
Weight difference in some situations.

#### How should this be manually tested?
checkout & yarn styleguide

#### Screenshots or example usage
Screen Shot 2019-12-17 at 10 33 43

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Totalizer/index.js [hunk 1]
```diff
@@ -68,11 +70,26 @@ class Totalizer extends PureComponent {
                   </div>
                 ) : (
                   <>
+                    {item.inverted && (
+                      <div className={`mb${invertedMargin}`}>
```
Comment (matheusps) [inline react/components/Totalizer/index.js]: Why not just 
```suggestion
                      <div className={`${item.inverted ? 'mb' : 'mt'}${invertedMargin}`}>
``` ?

Comment (matheusps) [reply react/components/Totalizer/index.js]: This way you can avoid repeating the code down below:

Comment (kevinch) [reply react/components/Totalizer/index.js]: Harder to read in my opinion and "you're already within" the `{item.inverted && ` condition.

Comment (matheusps) [reply react/components/Totalizer/index.js]: Yeah, it makes sense.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,10 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
+## [9.100.0] - 2019-12-17
+
+### Added
+
+- `inverted` prop to `Totalizer` component.
+
 ## [9.99.1] - 2019-12-17
 
+- Ts fixes of version `9.99.0`.
```
Comment (emersonlaurentino) [inline CHANGELOG.md]:
```suggestion
### Fixed

- Ts fixes of version `9.99.0`.
```

Comment (kevinch) [reply CHANGELOG.md]: Will do in a next PR right away.

**Pull request merged**